wgpu-core/Cargo.toml

```diff
@@ -128,6 +131,7 @@ indexmap.workspace = true
 log.workspace = true
 once_cell = { workspace = true, features = ["std"] }
 parking_lot.workspace = true
+portable-atomic = { workspace = true, optional = true }
```

Comment:
This and `wgpu-hal` should still be gated behind `[target.'cfg(not(target_has_atomic = "64"))'.dependencies]` so you need _both_ the feature and the target property to get the dependency.